script.js

```diff
@@ -1,4 +1,181 @@
+//Roll the dice and put them in order to make the highest number possible. If you roll a 4 and an 6, for example, your best answer would be 64. Using 3 dice, a roll of 3, 5 and 2 should give you 532, and so on. Write down your answer, pass the dice, and challenge the next player to Beat That!
+
+////Game State for stages of the gameplay
+var gameState = "";
+
+//Global variable for number of players and initialize player 1
+var numberOfPlayers = 0;
+var playerNumber = 1;
+
+//Game state for highest or lowest combined score
+var gameMode = "";
```

Comment:
It would be good to start off with some SCREAM_CASE constants! So we know at glance all the possible game modes eg.
```

var PLAYER_SELECTION_ MODE = "players selection"
var DICE_SELECTION_ MODE = "dice selection"
var CHECK_WINNER_MODE = "check winner"
```

Then in the main function
`if (gameState == PLAYER_SELECTION_ MODE) {`